doozer/doozerlib/cli/scan_sources.py

```
@@ -81,6 +82,40 @@ async def run(self):
+        self.skipped_image_dgks.add(image_meta.distgit_key)
+        self.runtime.logger.warning("[%s] %s", image_meta.distgit_key, issue)
+        self.issues.append({"name": image_meta.distgit_key, "issue": issue})
+
+    def _prepare_image_for_scan(self, image_meta: ImageMetadata, phase: str) -> bool:
+        if image_meta.distgit_key in self.skipped_image_dgks:
+            return False
+        if not self._is_meta_enabled(image_meta):
+            return False
+        try:
+            image_meta.ensure_canonical_builders_resolved()
+        except Exception as err:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)